tutorials/polkadot-sdk/build-a-blockchain/simulate-a-network.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Simulate a Network
+description: Create a private blockchain network with authorized validators using Polkadot-SDK. Start nodes, connect peers and verify block production.
+---
+
+# Simulate a Network
+
+## Introduction
+
+This tutorial introduces you to the process of initiating a private blockchain network with a set of authorized validators. The [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} implements an authority consensus model to regulate block production.
+
+In this model, the creation of blocks is restricted to a predefined list of authorized accounts, known as "authorities," who operate in a round-robin fashion. 
+
+To demonstrate this concept, you'll simulate a network environment using two nodes running on a single computer, each configured with different accounts and keys. Throughout this tutorial, you'll gain practical insight into the functionality of the authority consensus model by observing how these two predefined accounts, serving as authorities, enable the nodes to produce blocks.
+
+By completing this tutorial, you will accomplish the following objectives:
+
+- Start a blockchain node using a predefined account
+- Learn the key command-line options used to start a node
+- Determine if a node is running and producing blocks
+- Connect a second node to a running network
+- Verify peer computers produce and finalize blocks
+
+## Prerequisites
+
+- Installed and configured Rust on your system. For detailed instructions on installing Rust and setting up your development environment, refer to the [Installation]() guide.
```

Comment:
```suggestion
- Installed and configured Rust on your system. Refer to the [Installation]() guide for detailed instructions on installing Rust and setting up your development environment.
```
I think this is clearer, however feel free to decide based on your own taste for this one